hack/test-go.sh

```diff
@@ -135,16 +135,21 @@ if [[ $iterations -gt 1 ]]; then
 fi
 
 if [[ -n "${1-}" ]]; then
-  covdir="/tmp/k8s_coverage/$(kube::util::sortable_date)"
-  kube::log::status "Saving coverage output in '${covdir}'"
+  cover_params=""
+  if [[ -n "${KUBE_COVER}" ]]; then
+    covdir="/tmp/k8s_coverage/$(kube::util::sortable_date)"
+    kube::log::status "Saving coverage output in '${covdir}'"
+    mkdir -p "${covdir}/${arg}"
+    eval "cover_params=(${KUBE_COVER} -coverprofile=\"${covdir}/${arg}/coverage.out\")"
```

Comment:
@filbranden good suggestion; I realize the above also has a bug where the coverage reports are created incorrectly.